stablehlo/transforms/CMakeLists.txt

```diff
@@ -16,18 +16,6 @@ set(LLVM_TARGET_DEFINITIONS Passes.td)
 mlir_tablegen(Passes.h.inc -gen-pass-decls)
 add_public_tablegen_target(PassesIncGen)
 
-add_mlir_dialect_library(StablehloTypeConversion
-  PARTIAL_SOURCES_INTENDED
-  TypeConversion.cpp
```

Comment:
How about we move the definitions of these functions from the .h file to the .cpp file? The header is already 200+ lines of code.